clappr/clappr-ios
# Seek problems tvos

# Goal
This PR solves the problem where in tvos, trying to seek by pressing the rightArrow key and then playing had no effect.

# How to test
1. Open the sample app
2. Pause the video
3. Seek the video
4. The video should start playing where the seek has stopped

Test multiple ways of pausing the video, i.e. play/pause button or select button (pressing the touchpad)
Also try multiple ways of seeking, i.e. by pressing the right arrow in the keyboard or swiping right the remote control.

If you are testing before/after, make sure you're using the simulator with tvos 12.1 (xcode 10.1)

# Tech notes
The previous implementation with a UITapRecognizer was conflicting with existing tap recognizers in the viewController.
